### PR TITLE
Fix different height of actions with custom icon

### PR DESCRIPTION
**Pull request by raimund-schluessler**
With #619 the actions menu causes a parent element to have a larger height than without a custom icon (custom icon on second Actions):
Screenshot_2019-10-03 Inventar - Nextcloud

This PR fixes this issue:
Screenshot_2019-10-03 Inventar - Nextcloud(1)

### CODE REVIEW

**Comment by skjnldsv** [comment]
We should probabyl add an example on the docs for this :)

**Comment by skjnldsv** [comment]
> Specify the icon to be shown as placeholder when more than one action is inside the actions component

Also the prop comment is maybe not clear that the default is the three dot menu.
We should maybe not encourage people to change the icon, but in very specific cases only. Could you take care of this as well? :)
What do you think? cc @georgehrke

**Comment by raimund-schluessler** [comment]
Added an example and hopefully improved the docs.

File: src/components/Actions/Actions.vue [hunk 1]
```diff
@@ -42,6 +42,16 @@ https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-action
 	<ActionLink icon="icon-external" title="Link" href="https://nextcloud.com" />
 </Actions>
 ```
+
+### Multiple actions with custom icon
+
+```
+<Actions :default-icon="'icon-edit'">
```
Comment (skjnldsv) [inline src/components/Actions/Actions.vue]:
```suggestion
<Actions default-icon="icon-edit">
```

**Pull request merged**